spring-boot-starter-camunda/src/test/java/io/camunda/zeebe/spring/client/config/ZeebeClientStarterAutoConfigurationTest.java

```diff
@@ -0,0 +1,81 @@
+package io.camunda.zeebe.spring.client.config;
```

Comment:
let's change all packages from `io.camunda.zeebe.spring.client` to `io.camunda.zeebe.client.spring` I would argue having `spring` as subpackage on the client better represents this is an addition to the zeebe client